docs/organizations/managing-security-and-risk.md

```diff
@@ -32,29 +32,85 @@ Each of these areas displays a count of matching items and a **Review** button t
 
 The Security and Risk Management item list displays a list of items, filtered by status and sorted by due date ascending.
 
-To access the item list, access the [dashboard](#dashboard) and click the **Review** button under the area of interest, based on the desired filtering.
+To access the item list, access the [dashboard](#dashboard) and click the **Review** button under an area of interest, based on the desired filtering.
 
-To find out more about an item, click its **Details** column. This redirects you to a filtered view of the **Repository issues** page, showing only the item of interest.
+To open the originating issue of an item, click the **Details** column of the item of interest.
 
 ![Security and Risk Management item list](images/security-risk-management-item-list.png)
 
+## Managing integrations
+
+Integrations let you detect security vulnerabilities across a number of platforms and display them all as security items on the unified item list.
+
+The following section explains how to integrate with each of the supported platforms.
+
+=== "Codacy"
+
+    The Codacy integration is active by default and can't be deactivated.
```

Comment:
It's weird to call "integration" if the data is in the same system. Do we (as a product) want to keep calling this an integration? 🤔

Reply:
That's a good point. I'm rewording this not to mention "integration", but I still believe it should appear together with other integrations since they're all conceptually the same item, i.e. "data sources".